content/docs/rfcs/19/README.md

```diff
@@ -41,17 +40,28 @@ message PushRPC {
 }
 ```
 
-## Message relaying
+## Message Relaying
 
-Nodes that respond to `PushRequests` MUST relay this via [11/WAKU2-RELAY](/spec/11) protocol on the specified `pubsub_topic`.
+Nodes that respond to `PushRequests` MUST either
+relay the encapsulated message via [11/WAKU2-RELAY](/spec/11) protocol on the specified `pubsub_topic`,
+or forward the `PushRequest` via 19/LIGHTPUSH on a [44/WAKU2-DANDELION](https://rfc.vac.dev/spec/44/) stem.
 If they are unable to do so for some reason, they SHOULD return an error code in `PushResponse`.
 
-## Security considerations
+## Security Considerations
 
-Since this can introduce amplification factor, it is RECOMMENDED for the node relaying to the rest of the network to extra precaution. This can be done by various forms of rate limiting, or by using [18/WAKU2-SWAP](/spec/18) to account for the service provided.
+Since this can introduce an amplification factor, it is RECOMMENDED for the node relaying to the rest of the network to take extra precautions.
+This can be done by rate limiting via [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17/),
+or by using [18/WAKU2-SWAP](/spec/18) to account for the service provided.
```

Comment:
I'd probably remove SWAP reference as this is currently not part of our offering or plan